feature/savings-account/src/commonMain/kotlin/org/mifos/mobile/feature/savingsaccount/SavingsAccountScreen.kt

```diff
@@ -0,0 +1,329 @@
+                        ),
+                        style = MifosTypography.labelMedium,
+                        color = MaterialTheme.colorScheme.secondary,
+                    )
+                }
+
+                Row(
+                    horizontalArrangement = Arrangement.spacedBy(DesignToken.spacing.largeIncreased),
+                ) {
+                    Icon(
+                        modifier = Modifier
+                            .clickable {}
+                            .size(20.dp),
+                        imageVector = MifosIcons.SearchNew,
+                        contentDescription = "search",
+                    )
+                    Icon(
+                        modifier = Modifier
+                            .clickable {}
+                            .size(20.dp),
+                        imageVector = MifosIcons.Filter,
+                        contentDescription = "filter",
+                    )
+                }
+            }
+
+            Spacer(modifier = Modifier.height(DesignToken.spacing.medium))
+
+            HorizontalDivider(
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .height(0.99997.dp),
+            )
+
+            LazyColumn(
+                modifier = Modifier
+                    .fillMaxSize()
+                    .weight(1f),
+            ) {
+                item {
+                    Spacer(modifier = Modifier.height(DesignToken.spacing.small))
+                }
+                items(accountCards) { account ->
+                    val color = when (account.statusText) {
+                        "Progress" -> AppColors.customEnable
+                        "Pending" -> AppColors.customYellow
+                        "Closed" -> MaterialTheme.colorScheme.error
+                        else -> MaterialTheme.colorScheme.onSurface
+                    }
+
+                    SavingsAccountCard(
+                        accountId = account.accountId,
+                        accountNumber = account.accountNumber,
+                        accountType = account.accountType,
+                        accountStatus = if (account.statusText == "Progress") {
+                            account.statusValue ?: ""
+                        } else {
+                            account.statusText
+                        },
+                        accountStatusColor = color,
+                        onAccountClick = {},
+                    )
+                }
+            }
+        }
+    }
+}
+
+data class AccountCardData(
+    val accountId: Long,
+    val accountNumber: String,
+    val accountType: String,
+    val statusText: String,
+    val statusValue: String? = null,
+)
+
+val accountCards = listOf(
+    AccountCardData(
+        accountId = 1L,
+        accountNumber = "2689-7897-6666",
+        accountType = "Wallet Account",
+        statusText = "Progress",
+        statusValue = "$ 23,315,500",
```

Comment:
Just a questions is these predefined `accountCards` are just for demonstrating purpose it will be removed later right?